Dockerfile

```diff
@@ -1,28 +1,10 @@
-## Builder part is quite heavy as it depends on the Go toolchain
-FROM golang:1.11-alpine AS builder
-LABEL maintainer="BOSCoin Developers <devteam@boscoin.io>"
-
-# You probably don't need to change this
-ARG BUILD_MODE="install"
-ARG BUILD_ARGS=''
-ARG BUILD_PKG="./..."
-
-RUN apk add --no-cache git openssh gcc musl-dev linux-headers
-RUN go get github.com/ahmetb/govvv
-
-#golang:alpine set $GOPATH to `/go`
-COPY ./ /sebak-build
-WORKDIR /sebak-build
-
-RUN go $BUILD_MODE $BUILD_ARGS -ldflags="$(govvv -pkg boscoin.io/sebak/lib/version -flags)" -v $BUILD_PKG
-
 ## This one is much more lightweight
 FROM alpine:latest AS runner
 
 RUN apk --no-cache add ca-certificates      # For SSL requests
 COPY docker/entrypoint.sh /sebak/entrypoint.sh
 COPY docker/sebak.* /sebak/
-COPY --from=builder /go/bin/sebak /sebak/
+COPY --from=sebak:builder /go/bin/sebak /sebak/
```

Comment:
What I don't like about this setup is that it forces the user to use a specific tag.
If someone wants to use, e.g. `sebak-stable`, this will break, but the user might not see it if there's already a `sebak` tagged.
It also relies more on user's virtue to build `sebak` before building this, right ?